frontend/src/pages/control-center/ControlCenter.jsx

```diff
@@ -76,9 +76,11 @@ const ControlCenter = () => {
             </Card>
           </Col>
         ))}
-        <Col span={24}>
-          <PanelApprovals />
-        </Col>
+        {authUser.role_detail.page_access.includes("approvals") && (
+          <Col span={24}>
+            <PanelApprovals />
+          </Col>
+        )}
```

Comment:
Where is the PanelDataUpload? also selectedPanels above this line should be filtered